webui/react/src/pages/F_ExpList/glide-table/GlideTable.tsx

```diff
@@ -677,6 +677,12 @@ export const GlideTable: React.FC<GlideTableProps> = ({
           case V1LocationType.CUSTOMMETRIC:
             dataPath = `bestTrial.summaryMetrics.${currentColumn.column}`;
             break;
+          case V1LocationType.RUN:
+            dataPath = `run.${currentColumn.column}`;
+            break;
+          case V1LocationType.RUNHYPERPARAMETERS:
+            dataPath = `run.hyperparameters.${currentColumn.column.replace('hp.', '')}`;
+            break;
```

Comment:
consider throwing an error here instead -- this table should only get `ExperimentWithTrial` objects, so there's no `run` object in the response. this logic will be table-specific in the future as well.